### ISSUE TITLE
Extracting csv file data into telegram through python cmd

### ISSUE THREAD

**Issue by bigsmallninja** [thephpleague/csv#471]
_(Fill in the relevant information below to help triage your issue.)_

|    Q        |   A
|------------ | ------
| Version     | x.y.z

## Question

_(Please explain in plain english your question)_

## Checks before submitting

* [x] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/thephpleague/csv/issues)
* [x] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/thephpleague/csv/pulls)
* [x] I have read, search and not found the information on the [documentation website](https://csv.thephpleague.com).
* [x] I have read, search and not found the information on PHP related forums and/or websites.
* [x] This issue is about 1 question around the package with no business or domain specific logic related to a specific situation.
* [x] The question has a descriptive title. For example:  "Can I use the library with compressed CSV documents ?".

**Comment by bigsmallninja** [comment]
so once i run the code im left with 

Traceback (most recent call last):
  File "C:\Users\HOME\Desktop\telegramscraper-main\tsadder.py", line 153, in <module>
    with open('members\\members.csv', 'w', encoding='UTF-8') as f:
PermissionError: [Errno 13] Permission denied: 'members\\members.csv'

C:\Users\HOME\Desktop\telegramscraper-main>

anyone help ?

**Comment by nyamsprod** [comment]
@bigsmallninja This is not related to league/csv at all, so I'll close the issue as there's nothing to fix.
